CRITICAL - 2024-04-04 08:46:47 --> Error: Undefined constant "App\Models\usuario"
in APPPATH\Models\Usuario.php on line 12.
 1 APPPATH\Controllers\Login.php(42): App\Models\Usuario->login([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-04 08:47:36 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'admin'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), '', 3308, '', 0)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#5 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-04-04 08:47:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuario`', [], false)
 3 APPPATH\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Login.php(42): App\Models\Usuario->login([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-04 08:58:35 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'admin'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#5 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-04-04 08:58:35 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuario`', [], false)
 3 APPPATH\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Login.php(42): App\Models\Usuario->login([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-04 09:19:22 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'admin'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#5 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-04-04 09:19:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuario`', [], false)
 3 APPPATH\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Login.php(42): App\Models\Usuario->login([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-04 09:19:29 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'admin'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#5 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-04-04 09:19:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuario`', [], false)
 3 APPPATH\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Login.php(42): App\Models\Usuario->login([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-04 15:12:00 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'admin'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\tfg\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tfg\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tfg\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tfg\app\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tfg\app\Controllers\Login.php(42): App\Models\Usuario->login(Array)
#5 C:\xampp\htdocs\tfg\system\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
#6 C:\xampp\htdocs\tfg\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\tfg\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tfg\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-04-04 15:12:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuario`', [], false)
 3 APPPATH\Models\Usuario.php(12): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Login.php(42): App\Models\Usuario->login([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->iniciarSesion()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
